tests/basic/function/Sample-Docker

```diff
@@ -0,0 +1,15 @@
+FROM icr.io/quantum-public/qiskit-serverless/ray-node:0.14.0-py310
```

Comment:
depending on when this merges, we may need to drop the python version
```suggestion
FROM icr.io/quantum-public/qiskit-serverless/ray-node:0.14.0
```

Reply:
From my last review here Aki, Paul I would like to keep it if both of you agree 👍 

https://github.com/Qiskit/qiskit-serverless/pull/1420#issuecomment-2250448999

Reply:
I just saw your comment, Paul. I couldn't dedicate too much time to process everything, sorry. Tomorrow I'm going to read calmly everything and maybe we can connect to discuss it a bit. We can always do this change in that PR I assume 😄